docs/hpc/containers.md

```diff
@@ -109,18 +112,36 @@ From: ubuntu:20.04
 
 Your `requirements.txt` file will contain all the [necessary packages](https://learnpython.com/blog/python-requirements-file/) for your Python container.
 
-(To add a package to the existing Python container, you can access the recipe file named `python.recipe`. On line 61 is a line instructing the container to install a number of Python modules from the `requirements.txt` file. If you do not see a package required for your code simply add your package name to this file and rebuild the container image in a Linux environment with root access.
+(To add a package to the existing Python container, you can access the recipe file named `python.recipe`. On line 61 is a line instructing the container to install a number of Python modules from the `requirements.txt` file. If you do not see a package required for your code simply add your package name to this file and rebuild the container image in a Linux environment.
 
-To add a package to the existing R container, you can access the recipe file named `R.recipe`. On line 85 is a line instructing R to install a number of packages. If you do not see a package required for your code simply add your package name to this list and rebuild the container image in a Linux environment with root access.)
+To add a package to the existing R container, you can access the recipe file named `R.recipe`. On line 85 is a line instructing R to install a number of packages `apt-get install --y --no-install-recommends`. If you do not see a package required for your code simply add your package name to this list and rebuild the container image in a Linux environment.
 
-Once you've created this file, execute the following command:
+Once you've created this file, load singularity and execute the following command:
 
 ```
-sudo singularity build container.recipe container.simg
+singularity build --remote <container-name>.simg <container-name>.recipe 
 ```
 
-This will create a new singularity container with all your required Python packages.
+You may need to create a singularity account to execute this command. Sign up at [Sylabs](https://cloud.sylabs.io). If you see that you need a token or your token is expired, you can generate a new one [here](https://cloud.sylabs.io/auth/tokens). Then you should be able to login:
+```
+singularity remote login
+```
+...with your token and execute the build command above to create a new singularity container "container.simg" with all your required Python packages.
```

Comment:
Should this be "container-name.simg" to match the above?

Reply:
Yes